subsys/net/lib/nrf_cloud/Kconfig.nrf_cloud_pgps

```diff
@@ -93,4 +93,26 @@ config NRF_CLOUD_PGPS_SOCKET_RETRIES
 	help
 	  This sets the maximum number of times to retry a download.
 
+config NRF_CLOUD_PGPS_PARTITION
+	bool "Define P-GPS subsystem partition"
+	default y if !MEMFAULT && SECURE_BOOT
+	help
```

Comment:
It would be helpful if you could add a few lines here explaining why this configuration cannot be used together with MEMFAULT and SECURE_BOOT.

Reply:
I replaced `SECURE_BOOT` with `BUILD_S1_VARIANT` since that is the more correct value.

I've improved the help as well.

Reply:
I tested this again with it rebased onto the latest main, and now it builds fine if all 3 things are defined -- P-GPS partition, Memfault, and MCUboot FOTA (SECURE_BOOT + BUILD_S1_VARIANT).

So there is no need to qualify this based on Memfault.